### DISCUSSION TITLE
Comparison with `eslint-plugin-jest` rules

### DISCUSSION THREAD

**Discussion by unlikelyzero** [playwright-community/eslint-plugin-playwright#90] [General]
This Discussion serves as a living tracker to capture all available eslint-plugin-jest rules and document their applicability and implementation status.

Current as of:
eslint-plugin-jest v26.8.7
eslint-plugin-playwright v0.11.1

| eslint-plugin-jest | eslint-plugin-playwright | Notes |
| --- | --- | --- |
| [consistent-test-it](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/consistent-test-it.md) | N/A | Not applicable to Playwright |
| [expect-expect](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md) | ❌   | Applicable. No Issue filed |
| [max-expects](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/max-expects.md) | ❌   | Applicable. No issue filed |
| [max-nested-describe](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/max-nested-describe.md) | ✅   | Implemented |
| [no-alias-methods](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-alias-methods.md) | ?   | I'm not sure if this is applicable |
| [no-commented-out-tests](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-commented-out-tests.md) | ❌   | Applicable. No Issue Filed. |
| [no-conditional-expect](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md) | ❌   | Applicable. No Issue Filed. |
| no-conditional-in-test | ✅   | Implemented |
| [no-deprecated-functions](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-deprecated-functions.md) | ❌   | Applicable. No Issue Filed. |
| [no-disabled-tests](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-disabled-tests.md) | ✅   | Covered in other no-skipped-tests |
| [no-done-callback](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-done-callback.md) | N/A | Not applicable |
| [no-duplicate-hooks](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-duplicate-hooks.md) | ❌   | Applicable. No Issue Filed. |
| [no-export](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-export.md) | N/A | Not applicable (I think) |
| [no-focused-tests](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-focused-tests.md) | ✅   | Implemented |
| [no-hooks](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-hooks.md) | ❌   | Applicable. No Issue Filed. |
| [no-identical-title](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-identical-title.md) | ❌   | Applicable. No Issue Filed. |
| [no-if](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-if.md) | N/A | Deprecated |
| [no-interpolation-in-snapshot](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-interpolation-in-snapshots.md) | N/A | Not applicable (I think) |
| [no-jasmine-globals](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-jasmine-globals.md) | N/A | Not applicable (I think) |
| [no-jest-import](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-jest-import.md) | ❌   | Applicable. No Issue Filed. |
| [no-large-snapshots](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-large-snapshots.md) | ❌   | Applicable. No Issue Filed. |
| [no-mocks-import](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-mocks-import.md) | N/A | Not applicable (I think) |
| [no-restricted-matchers](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-restricted-matchers.md) | ✅   | Implemented |
| [no-standalone-expect](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-standalone-expect.md) | ❌   | Applicable. No Issue Filed. |
| [no-test-prefixes](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-test-prefixes.md) | N/A | Not Applicable. |
| [no-test-return-statement](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-test-return-statement.md) | N/A | Not Applicable. |
| [prefer-called-with](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-called-with.md) | N/A | Not Applicable. |
| [prefer-comparison-matcher](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-comparison-matcher.md) | ❌   | Applicable. No Issue Filed. |
| [prefer-equality-matcher](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-equality-matcher.md) | ❌   | Applicable. No Issue Filed. |
| [prefer-expect-assertions](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-expect-assertions.md) | N/A | Not applicable (I think) |
| [prefer-expect-resolves](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-expect-resolves.md) | N/A | Not applicable (I think) |
| [prefer-hooks-in-order](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-hooks-in-order.md) | ❌   | Applicable. No Issue Filed. |
| [prefer-hooks-on-top](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-hooks-on-top.md) | ❌   | Applicable. No Issue Filed. |
| [prefer-lowercase-title](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-lowercase-title.md) | ✅   | Implemented |
| [prefer-snapshot-hint](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-snapshot-hint.md) | N/A | Not applicable (I think) |
| [prefer-mock-promise-shorthand](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-mock-promise-shorthand.md) | N/A | Not applicable (I think) |
| [prefer-spy-on](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-spy-on.md) | N/A | Not applicable |
| [prefer-strict-equal](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-strict-equal.md) | ❌   | Applicable. No Issue Filed. |
| [prefer-to-be](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-to-be.md) | N/A | Not Applicable. |
| [prefer-to-contain](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-to-contain.md) | N/A | Not Applicable. |
| [prefer-to-have-length](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-to-have-length.md) | ✅   | Implemented |
| [prefer-todo](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-todo.md) | ❌   | Applicable. [Feature Filed](https://github.com/microsoft/playwright/issues/10918). No matching Issue filed |
| [require-hook](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/require-hook.md) | ❌   | Applicable. No Issue Filed. |
| [require-to-throw-message](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/require-to-throw-message.md) | N/A | Not Applicable |
| [require-top-level-describe](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/require-top-level-describe.md) | ✅   | Implemented |
| [unbound-method](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/unbound-method.md) | ❌   | Applicable. No Issue Filed. |
| [valid-describe-callback](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-describe-callback.md) | ❌   | Applicable. No Issue Filed. |
| [valid-expect-in-promise](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-expect-in-promise.md) | ❌   | Applicable. No Issue Filed. |
| [valid-expect](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-expect.md) | ✅   | Implemented |
| [valid-title](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-title.md) | ❌   | Applicable. No Issue Filed. |

**Comment by mskelton** [comment]
@unlikelyzero Quite a few of these rules have been implemented or issues have been opened. If you would like to update this table that would be much appreciated!

**Reply by unlikelyzero** [reply]
Will do today.
